### ISSUE TITLE
Does not mock the provider functions

### ISSUE THREAD

**Issue by greeze** [launchdarkly-labs/jest-launchdarkly-mock#3]
The provider functions `asyncWithLDProvider` and `withLDProvider` are not mocked, but are exported from 'launchdarkly-react-client-sdk'. One approach might be something along these lines:

```javascript
jest.mock('launchdarkly-react-client-sdk', () => {
  return {
    ...jest.requireActual('launchdarkly-react-client-sdk'),
    useLDClient: jest.fn(),
    useFlags: jest.fn(),
    withLDConsumer: jest.fn(),
  }
})
```

**Comment by leftmostgeek** [comment]
I also Just ran into this issue except I'm using `LDProvider` is it possible to implement mocks for the providers in this lib?

**Comment by SHND** [comment]
It seems that this is not still published on npm. Do you know when is it possible to have this on npm?

**Comment by dv297** [comment]
@yusinto I'm wondering if #15 was lost during some transition. Not totally sure about the branch set up, but it looks like that PR was merged to master but now this repo only has a main branch, and this change was not published as part of the 1.0.0 release.

**Comment by debaraj-barua** [comment]
I was wondering the same thing as @dv297, additionally, I was wondering if the `withLDConsumer` is mocked as well.
